pages/common/less.md

```diff
@@ -22,6 +22,10 @@
 
 `?{{something}}`
 
+- Tail currently opened file:
```

Comment:
How about - `Follow the output of the currently opened file`

Reply:
Done